.github/workflows/runOnGitHub.yml

```diff
@@ -0,0 +1,29 @@
+# .github/workflows/runOnGitHub.yml
+# GitHub Actions documentation
+# => https://docs.github.com/en/actions
+name: runOnGitHub
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ master, main ]
+  pull_request:
+    branches: [ master, main ]
```

Comment:
You can trigger the CI on any pull request branches not just named ("master" and "main")